ScarletEngine/include/Core/Logger.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <cstdio>
+#include <cassert>
+#include <ctime>
+
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+
+namespace ScarletEngine
+{
+	enum class LogLevel
```

Comment:
Very happy you set this up for us. It's very similar to the ones at ubi :)

Reply:
Hahaha yeah it's slightly inspired by the `UE_LOG` macro in Unreal Engine which has a very similar syntax but provides a bit more functionality which I think I might try to add in a future version.